web/client/utils/GeoStoryUtils.js

```diff
@@ -205,6 +207,28 @@ export const getDefaultSectionTemplate = (type, localize = v => v) => {
                 }
             ]
         };
+    case SectionTypes.BANNER:
```

Comment:
The section banner should be similar to the title without the content text. So we should remove all not needed properties.

eg:
```js
case SectionTypes.BANNER:
    return {
        id: uuid(),
        type: SectionTypes.BANNER,
        title: localize("geostory.builder.defaults.titleBanner"),
        cover: false,
        contents: [{
            id: uuid(),
            background: {
                fit: 'cover',
                size: 'full',
                align: 'center'
            }
        }]
    };
```